tools/pepsi/src/analyze.rs

```diff
@@ -51,3 +59,100 @@ pub async fn analyze(arguments: Arguments) -> Result<()> {
 
     Ok(())
 }
+
+fn to_string_pretty(value: &impl ToWriterPretty) -> Result<String, fmt::Error> {
+    let mut string = String::new();
+    value.to_writer_pretty(&mut string)?;
+    Ok(string)
+}
+
+trait ToWriterPretty {
+    fn to_writer_pretty(&self, f: &mut impl Write) -> fmt::Result;
+}
+
+impl ToWriterPretty for Instance {
+    fn to_writer_pretty(&self, f: &mut impl Write) -> fmt::Result {
+        write!(f, "{}", self.name)
+    }
+}
```

Comment:
`f`?! Please don't abbreviate. Also below.

Reply:
Done. First expanded to `formatter` but realized that actually doesn't fit here, likely a leftover from a `Display` implementation.
Named `writer` instead